### DISCUSSION TITLE
Is it possible to individually disable dependency updates?

### DISCUSSION THREAD

**Discussion by noahtallen** [renovatebot/renovate#12768] [Request Help] [answered]
We have a monorepo with many packages. We would like to disable updates for a peer dependency for just one of these packages. (This package needs to support an older version of react than our other packages do.)

Is it possible to have renovate stop updating this react peer dep in this one case? We would like react peer dependencies to be updated in general, so we don't want to turn it off altogether.

**Comment by viceice** [comment, marked as answer]
Yes, use a package rule with package name and path match.

**Reply by noahtallen** [reply]
Thanks! Here's my attempt:

```json
{
	"packageRules": [
		{
			"extends": "monorepo:react",
			"prPriority": 1,
			"ignorePaths": [ "packages/interpolate-components" ]
		},
		{
			"extends": [ "monorepo:react", ":disablePeerDependencies" ],
			"prPriority": 1,
			"includePaths": [ "packages/interpolate-components" ]
		}
	]
}
```
